cmd/copyEnumeratorInit.go

```diff
@@ -497,16 +497,19 @@ func (cca *cookedCopyCmdArgs) makeEscapedRelativePath(source bool, dstIsDir bool
 		if source {
 			relativePath = ""
 		} else {
+			if object.versionID != "" {
+				relativePath += "/" + object.versionID
+			}
```

Comment:
This does not look correct, I thought the blob name would become blobname.versionid. Also, the cx might have specified a folder as the destination.

Reply:
Awaiting response from StgExp folks.

Reply:
Done. Going ahead with "-" in the name. 
"2020-08-05T06-59-20.9890140Z-myblob"